backendless_chef.yaml

```diff
@@ -578,11 +578,9 @@ Resources:
         Version: "2012-10-17"
         Statement:
           - Effect: "Allow"
-            Principal: "*"
+            Principal:
+              AWS: !GetAtt ChefRole.Arn
```

Comment:
I'm wondering if this should be:
```
!If [CreateChefServerIamRole, !GettAtt ChefRole.Arn, !Ref ChefServerIamRole]
```

but now I'm wondering what you're even supposed to pass in as `ChefServerIamRole`  (is it an Arn? or just a role name?)